EventPlanning.DAL/Repository.cs

```diff
@@ -53,6 +54,21 @@ public bool SaveActivityType(ActivityType activityType)
             return true;
         }
 
+        public IQueryable<Event> GetEvents(string pattern)
+        {
+            return context.Event
+                .Where(e => e.Address.Contains(pattern)
+                         || e.Title.Contains(pattern)
+                         || e.Description.Contains(pattern)
+                         || e.UserId.ToString().Contains(pattern)
+                         || e.Id.ToString().Contains(pattern));
+        }
```

Comment:
Repository should not contain logic. Imagine we have a different database, which does not have this structure of model
Repository does not care how should it return data. It cares what data should it return: I return Events; I return Activities. But not which events (business logic should do that).